plugins/inputs/opcua/README.md

```diff
@@ -85,39 +105,61 @@ Plugin minimum tested version: 1.16
   ## namespace, this is used.
   # identifier_type =
   #
+  ## Default tags that are applied to every node in this group. Can be
+  ## overwritten in a node by setting a different value for the tag name.
+  ##   example: default_tags = { tag1 = "value1" }
+  # default_tags = {}
+  #
   ## Node ID Configuration.  Array of nodes with the same settings as above.
+  ## Use either the inline notation or the bracketed notation, not both.
+  #
+  ## Inline notation (default_tags not supported yet)
   # nodes = [
-  #  {name="", namespace="", identifier_type="", identifier=""},
-  #  {name="", namespace="", identifier_type="", identifier=""},
+  #  {name="node1", namespace="", identifier_type="", identifier=""},
+  #  {name="node2", namespace="", identifier_type="", identifier=""},
   #]
+  #
+  ## Bracketed notation
+  # [[inputs.opcua.group.nodes]]
+  #   name = "node1"
+  #   namespace = ""
+  #   identifier_type = ""
+  #   identifier = ""
+  #   default_tags = { tag1 = "override1", tag2 = "value2" }
+  #
+  # [[inputs.opcua.group.nodes]]
+  #   name = "node2"
+  #   namespace = ""
+  #   identifier_type = ""
+  #   identifier = ""
 
   ## Enable workarounds required by some devices to work correctly
   # [inputs.opcua.workarounds]
     ## Set additional valid status codes, StatusOK (0x0) is always considered valid
     # additional_valid_status_codes = ["0xC0"]
 
+  # [inputs.opcua.request_workarounds]
     ## Use unregistered reads instead of registered reads
     # use_unregistered_reads = false
 ```
 
 ## Node Configuration
 
-An OPC UA node ID may resemble: "n=3;s=Temperature". In this example:
+An OPC UA node ID may resemble: "ns=3;s=Temperature". In this example:
 
-- n=3 is indicating the `namespace` is 3
+- ns=3 is indicating the `namespace` is 3
 - s=Temperature is indicting that the `identifier_type` is a string and `identifier` value is 'Temperature'
 - This example temperature node has a value of 79.0
 To gather data from this node enter the following line into the 'nodes' property above:
 
-```shell
+```text
 {field_name="temp", namespace="3", identifier_type="s", identifier="Temperature"},
 ```
 
 This node configuration produces a metric like this:
 
 ```text
-opcua,id=n\=3;s\=Temperature temp=79.0,quality="OK (0x0)" 1597820490000000000
-
+opcua,id=ns\=3;s\=Temperature temp=79.0,quality="OK (0x0)" 1597820490000000000
```

Comment:
Did the namespace part of the id change from n to ns or was this a typo? If it changed, won't that cause problems for users with data saved from previous versions of telegraf?

Reply:
It was a typo.